packages/vue/src/tracing.ts

```diff
@@ -83,7 +83,7 @@ export const createTracingMixins = (options: TracingOptions): Mixins => {
         // Skip components that we don't want to track to minimize the noise and give a more granular control to the user
         const name = formatComponentName(this, false);
         const shouldTrack = Array.isArray(options.trackComponents)
-          ? options.trackComponents.includes(name)
+          ? options.trackComponents.indexOf(name) > -1
```

Comment:
Couldn't get tests to work because there was a type error here. I could have messed with tsconfigs but there is an upside of doing indexOf() instead of includes() which is backwards compatibility. I think includes is ES6 only.

Reply:
Lol, coincidence? https://github.com/getsentry/sentry-javascript/pull/5389